sdk/communication/azure-communication-identity/tests/utils.py

```diff
@@ -18,4 +19,24 @@ def is_token_expiration_within_allowed_deviation(
     expected_seconds = expected_token_expiration.total_seconds();
     time_difference = abs(expected_seconds - token_expiration_in_seconds)
     allowed_time_difference = expected_seconds * allowed_deviation
-    return time_difference < allowed_time_difference
\ No newline at end of file
+    return time_difference < allowed_time_difference
+
+def generate_teams_user_aad_token(self):
```

Comment:
as we returned back the parent class, maybe better place for this method and skip_get_token_for_teams_user_test will be the ACSIdentityTestCase, instead of expecting any params they can reach to the class properties, wdyt?

Reply:
Good idea! I'll update